traffic_ops/app/lib/API/User.pm

```diff
@@ -69,7 +69,7 @@ sub index {
 
 	my $dbh;
 	if ( defined $username ) {
-		$dbh = $self->db->resultset("TmUser")->search( { username => $username }, { prefetch => [ { 'role' => undef } ], order_by => 'me.' . $orderby } );
+		$dbh = $self->db->resultset("TmUser")->search( { username => $username }, { prefetch => [ { 'role' => undef }, 'tenant' ], order_by => 'me.' . $orderby } );
```

Comment:
Will change, I was not sure why it used the  { 'role' => undef } in the first place so I kept it.
The weird thing is that I have changed the wrong line, I should have changed the line in the "else" clause as $username is not defined when calling /users route.  But, it still works without prefetch on the tenant.  not sure how.